tests/settings.py

```diff
@@ -64,7 +64,7 @@
     'LONG_VALUE': (long_value, 'some looong int'),
     'BOOL_VALUE': (True, 'true or false'),
     'STRING_VALUE': ('Hello world', 'greetings'),
-    'UNICODE_VALUE': (six.u('Rivière-Bonjour'), 'greetings'),
+    'UNICODE_VALUE': (u'Rivière-Bonjour', 'greetings'),
```

Comment:
It should be fine on 3.3+ , and I understand constance doesn't support 3.2 any more?  eg tox testing was removed https://github.com/jazzband/django-constance/commit/3fef92734927cefabccc9b7eab32207e8bbe306a

It seemed like with six.u() the test was sending wrongly encoded data before.